tests/test_tool_by_tool_walkthrough.py

```diff
@@ -42,7 +43,12 @@ def steps() -> tuple:
 @pytest.mark.slow
 def test_full_standaard_run(steps):
     step_1, step_3, step_4, step_5 = steps
-    run(step_1)
+    try:
```

Comment:
Ik begrijp je punt hier beter. Hier is het inlezen van de kaart een belangrijk onderdeel van de test. Dus dit ziet er inderdaad een beetje vreemd uit.

Persoonlijk zou ik dit op de volgende manier oplossen:
```python
IS_WINDOWS = os.name == "nt"

@pytest.mark.skipIf(IS_WINDOWS, "skip test on windows, because or drivers")
def full_standaard_run(steps)
```

Reply:
dat zou dan iets explicieter moeten zijn ;-)

Reply:
Ben hier wel fan van